string str_var
int64 num_var=0
